packages/@adobe/spectrum-css-temp/components/barloader/index.css

```diff
@@ -62,6 +68,11 @@ governing permissions and limitations under the License.
   flex-flow: row;
   justify-content: space-between;
   inline-size: auto;
+  min-width: fit-content;
+
+  .spectrum-BarLoader-track {
+    width: auto;
+  }
```

Comment:
while one of these seems needed, where is this coming from?
also, line 70 has `inline-size: auto`, which is `width: auto`, i'm surprised it's needed for sideLabel and for track? lets be consistent and use inline-size
we can also use inline-size anywhere we have width, so `min-width` -> `min-inline-size`

Reply:
So this line prevents the `long label, labelPosition: side` story from having a track larger than 192px. I appears the track auto width without the auto width in the side label parent works just fine, so I'll remove that. And I'll update to inline-size. :)